core-war/src/main/webapp/wopi/jsp/wopi.jsp

```diff
@@ -0,0 +1,325 @@
+<fmt:message var="enable" key="wopi.action.enable"/>
+<fmt:message var="disable" key="wopi.action.disable"/>
+<fmt:message var="disableConfirm" key="wopi.action.disable.confirm"/>
+<fmt:message var="disableConfirmHelp" key="wopi.action.disable.confirm.help"/>
+<fmt:message var="revokeSelected" key="wopi.action.selection.revoke"/>
+<fmt:message var="revokeSelectedConfirm" key="wopi.action.selection.revoke.confirm"/>
+<fmt:message var="revokeAll" key="wopi.action.all.revoke"/>
+<fmt:message var="revokeAllConfirm" key="wopi.action.all.revoke.confirm"/>
+
+<fmt:message var="userNameLabel" key="wopi.user.name"/>
+<fmt:message var="userEditedFileLabel" key="wopi.user.editedFiles"><fmt:param value="${1}"/></fmt:message>
+<fmt:message var="userEditedFilesLabel" key="wopi.user.editedFiles"><fmt:param value="${2}"/></fmt:message>
+<fmt:message var="userNbEditedFilesLabel" key="wopi.user.editedFiles.nb"/>
+<fmt:message var="userLastEditionLabel" key="wopi.user.lastEdition"/>
+<fmt:message var="fileNameLabel" key="wopi.file.name"/>
+<fmt:message var="fileLocationLabel" key="wopi.file.location"/>
+<fmt:message var="fileEditorLabel" key="wopi.file.editors"><fmt:param value="${1}"/></fmt:message>
+<fmt:message var="fileEditorsLabel" key="wopi.file.editors"><fmt:param value="${2}"/></fmt:message>
+<fmt:message var="fileNbEditorsLabel" key="wopi.file.editors.nb"/>
+<fmt:message var="fileLastEditionLabel" key="wopi.file.lastEdition"/>
+
+<view:sp-page>
+  <view:sp-head-part>
+    <c:if test="${isEnabled}">
+      <style type="text/css">
+        #disableFormDialog .help {
+          font-size: 0.8em;
+        }
+        #dynamic-containers {
+          display: table;
+          width: 100%;
+        }
+        #dynamic-user-container,
+        #dynamic-user-container {
+          display: table-cell;
+        }
+        #dynamic-user-container {
+          width: auto;
+          padding-right: 5px;
+        }
+        #dynamic-file-container {
+          width: auto;
+          padding-left: 5px;
+        }
+        .tip-extra-info {
+          max-width: 1000px;
+        }
+      </style>
+    </c:if>
+    <script type="text/javascript">
+      <c:choose>
+      <c:when test="${isEnabled}">
+      let refreshTimer;
+      const userSelectionOptions = {
+        paramSelectedIds : 'selectedUserIds', paramUnselectedIds : 'unselectedUserIds'
+      };
+      const fileSelectionOptions = {
+        paramSelectedIds : 'selectedFileIds', paramUnselectedIds : 'unselectedFileIds'
+      };
+      let userArrayPaneAjaxControl;
+      let fileArrayPaneAjaxControl;
+      const userCheckboxMonitor = sp.selection.newCheckboxMonitor(
+          '#dynamic-user-container input[name=selection]');
+      const fileCheckboxMonitor = sp.selection.newCheckboxMonitor(
+          '#dynamic-file-container input[name=selection]');
+
+      function revokeSelected() {
+        jQuery.popup.confirm('${silfn:escapeJs(revokeSelectedConfirm)}', function() {
```

Comment:
why don't use in this whole JSP file our own notifying popin? (that is built currently upon jQuery popup)